projects/1.19/assets/farmers-utils/farmersutils/lang/zh_cn.json

```
@@ -0,0 +1,58 @@
+{
+  "itemGroup.farmersutils": "储存乐事",
+
+  "block.farmersutils.apple_bag": "袋装苹果",
+  "block.farmersutils.brown_mushroom_bag": "袋装棕色蘑菇",
+  "block.farmersutils.cocoa_bean_bag": "袋装可可豆",
+  "block.farmersutils.crimson_fungus_bag": "袋装绯红菌",
+  "block.farmersutils.egg_box": "盒装鸡蛋",
+  "block.farmersutils.glow_berry_bag": "袋装发光浆果",
+  "block.farmersutils.golden_apple_bag": "袋装金苹果",
+  "block.farmersutils.kelp_block": "海带块",
+  "block.farmersutils.red_mushroom_bag": "袋装红色蘑菇",
+  "block.farmersutils.sugar_cane_bale": "甘蔗捆",
+  "block.farmersutils.sweet_berry_bag": "袋装甜浆果",
+  "block.farmersutils.warped_fungus_bag": "袋装诡异菌",
+
+  "block.farmersutils.propelpearl_bag": "袋装枪药珠",
+  "block.farmersutils.propelplant_cane_bale": "枪药杆捆",
+
+  "block.farmersutils.crate": "板条箱",
+  "block.farmersutils.drying_rack": "风干棚",
+  "block.farmersutils.tray": "托盘",
+
+  "item.farmersutils.apple_slice": "苹果片",
+  "item.farmersutils.golden_apple_slice": "金苹果片",
+  "item.farmersutils.dried_apple_slice": "风干苹果片",
+  "item.farmersutils.dried_golden_apple_slice": "金风干苹果片",
+  "item.farmersutils.dried_beef": "牛肉干",
+  "item.farmersutils.dried_chicken": "鸡肉干",
+  "item.farmersutils.dried_cod_slice": "鳕鱼肉干",
+  "item.farmersutils.dried_mutton": "羊肉干",
+  "item.farmersutils.dried_porkchop": "猪肉干",
+  "item.farmersutils.dried_rabbit": "兔肉干",
+  "item.farmersutils.dried_salmon_slice": "蛙鱼肉干",
+  "item.farmersutils.boiled_egg": "熟鸡蛋",
+
+  "farmersutils.container.crate": "板条箱",
+
+  "config.farmersutils.general_settings": "基本设置",
+  "config.farmersutils.general_settings.enable_storages": "启用原版材料压缩成块存储",
+  "config.farmersutils.general_settings.enable_nethersdelight_storages": "启用 Nether's Delight 材料压缩成块存储",
```

要不要把下界乐事也翻出来？

```suggestion
  "config.farmersutils.general_settings.enable_nethersdelight_storages": "启用Nether's Delight材料压缩成块存储",
```
如果保留原文的话：在1.16或更高版本下，英文字母、阿拉伯数字等非中文文本无需在前后添加空格。
具体可见我们的 [翻译规范](https://cfpa.site/TransRules/) 中 **04.排版** 一节下。
诸如这类的问题请在您翻译的所有文件中全部查明并修改。